native/src/components/MalteHelpFormOffer.tsx

```diff
@@ -78,11 +85,25 @@ const MalteHelpFormOffer = ({
 
   const submit = handleSubmit(async _data => {
```

Comment:
Please remove the underscore, was just used to avoid linting complaining about an unused var.